assets/js/modules/idea-hub/components/dashboard/DashboardIdeasWidget/index.js

```diff
@@ -295,25 +301,23 @@ function DashboardIdeasWidget( props ) {
 				</div>
 
 				<div className="googlesitekit-idea-hub__body">
-					<div
-						className="googlesitekit-idea-hub__content"
-						aria-hidden={ activeTab !== 'new-ideas' }
```

Comment:
Is there a reason we removed the `aria-hidden` attribute and moved to a single wrapping `__content` div? It seems like this would be still relevant to keep.

Reply:
Restored this part as the `<Route>` elements weren't needed and we need to always render each tab's content to preserve the height.